docs/api/turn-by-turn/api-reference.md

```diff
@@ -494,6 +495,7 @@ The codes correspond to code returned from a particular [Valhalla project](https
 |140 | Action does not support multimodal costing |
 |141 | Arrive by for multimodal not implemented yet |
 |142 | Arrive by not implemented for isochrones |
+|143 | ignore_closure in costing and exclude_closure in search_filter cannot both be specified |
```

Comment:
This is the new exception type added which explains the error related to the new option.